pkg/backup/remap_crd_version_action.go

```diff
@@ -92,7 +98,8 @@ func (a *RemapCRDVersionAction) Execute(item runtime.Unstructured, backup *v1.Ba
 		if crd.Spec.Versions[0].Schema == nil || crd.Spec.Versions[0].Schema.OpenAPIV3Schema == nil {
 			log.Debug("CRD is a candidate for v1beta1 backup")
 
-			if err := setV1beta1Version(item); err != nil {
+			item, err = fetchV1beta1CRD(crd.Name, a.betaCRDClient)
```

Comment:
Not a comment on this line, but on line 78.
Is it possible that we might see unmarshalling errors in the json roundtrip if the item was a `v1beta1` CRD and we try to do the json conversion using the `v1` API?